memgpt/cli/cli_config.py

```diff
@@ -19,12 +19,17 @@
 
 
 def get_azure_credentials():
-    azure_key = os.getenv("AZURE_OPENAI_KEY")
-    azure_endpoint = os.getenv("AZURE_OPENAI_ENDPOINT")
-    azure_version = os.getenv("AZURE_OPENAI_VERSION")
-    azure_deployment = os.getenv("AZURE_OPENAI_DEPLOYMENT")
-    azure_embedding_deployment = os.getenv("AZURE_OPENAI_EMBEDDINGS_DEPLOYMENT")
-    return azure_key, azure_endpoint, azure_version, azure_deployment, azure_embedding_deployment
+    creds = dict(
```

Comment:
@sarahwooders refactored into returning dict, included some defaults for the embeddings env vars in case users forget to set them (they likely are the same as the model endpoints)